e2e/app-with-domain/app_with_domain_suite_test.go

```diff
@@ -25,7 +25,7 @@ var _ = BeforeSuite(func() {
 	copilotCLI, err := client.NewCLI()
 	cli = copilotCLI
 	Expect(err).NotTo(HaveOccurred())
-	appName = fmt.Sprintf("e2e-app-with-domain-%d", time.Now().Unix())
+	appName = fmt.Sprintf("%d", time.Now().Unix())
```

Comment:
if we get rid of this prefix how do we know this is the e2e test app for domain check?

Reply:
maybe `appName = fmt.Sprintf("e2e-domain-%d", time.Now().Unix())`

Reply:
On failures, the logs tell you which test failed: /github.com/aws/copilot-cli/e2e/app-with-domain/app_with_domain_test.go:64